README.md

```diff
@@ -560,6 +552,38 @@ def test_ts0121_signature(assert_signature_matches_quirk):
     assert_signature_matches_quirk(zhaquirks.tuya.ts0121_plug.Plug, signature)
 ```
 
+## Managing dependencies
+
+Running `uv sync` will install all packets using the locked version in the `uv.lock` file. The setup script will do this for you.
```

Comment:
I think the word `packet` should be replaced with `package`, there are quite a few occurrences in this section.